components/form/docs/form.md

```diff
@@ -0,0 +1,113 @@
+# Forms
+
+## d2l-form
+
+The `d2l-form` component can be used to build sections containing interactive controls that are validated and submitted as a group.
+
+It differs from the native HTML `form` element in 4 ways:
+1. It supports custom form elements made using the [`FormElementMixin`](./form-element-mixin.md) in addition to native form elements like `input`, `select` and `textarea`.
+1. Upon validation, it will display an error summary that contains error messages for any elements that failed validation.
+1. `d2l-form` elements can be nested. If a parent form is validated or submitted it will also trigger the corresponding action for descendent `d2l-form`'s unless they explicitly opt-out using. This means that a `d2l-form` will only pass validation if it and all of its nested descendants pass validation.
```

Comment:
```suggestion
1. `d2l-form` elements can be nested. If a parent form is validated or submitted it will also trigger the corresponding action for descendent `d2l-form`s unless they explicitly opt-out using `no-nesting`. This means that a `d2l-form` will only pass validation if it and all of its nested descendants pass validation.
```